packages/@react-spectrum/list/src/InsertionIndicator.tsx

```diff
@@ -0,0 +1,45 @@
+import {classNames} from '@react-spectrum/utils';
+import {ItemDropTarget} from '@react-types/shared';
+import listStyles from './listview.css';
+import {ListViewContext} from './ListView';
+import React, {useContext, useRef} from 'react';
+import {useVisuallyHidden} from '@react-aria/visually-hidden';
+
+interface InsertionIndicatorProps {
+  target: ItemDropTarget
+}
+
+export default function InsertionIndicator(props: InsertionIndicatorProps) {
```

Comment:
Noted that the drop indicator takes up space and causes layout shifting when moving it via keyboard (doesn't happen on mouse drop movement though). Haven't looked too deeply into why, but maybe the design of the ListView should be updated so there is room for that drop indicator? Or is something up with the keyboard drop indicator?

Reply:
Also noted that the announcement for dropping onto an item isn't as descriptive as when dropping in between two items. Targeting an item as the drop target will only say the name of the row but the announcement for dropping between the rows states "Insert between {row A} and {row B}"

Can be followup but would be nice to get a more descriptive announcment